website/docs/guides/upgrade.html.markdown

```diff
@@ -0,0 +1,158 @@
+---
+layout: "nsxt"
+page_title: "VMware NSX Terraform Provider NSX upgrade support"
+description: |-
+  The VMware NSX Terraform Provider NSX upgrade support
+---
+
+# NSX Provider for NSX Upgrade
+
+NSX Terraform Provider offers support for [upgrading NSX components](https://docs.vmware.com/en/VMware-NSX/4.1/upgrade/GUID-E04242D7-EF09-4601-8906-3FA77FBB06BD.html).
+
+The provider support consists of several resources and data sources which can perform a complete upgrade cycle when applied correctly.
+The NSX upgrade process upgrades the manager components of NSX, Edge appliances, and the NSX bits on the compute hosts.
```

Comment:
Another pedant suggestion, consider replacing with:

"The NSX Terraform provider support upgrades for NSX on ESX hosts, NSX Edge appliances, and NSX manager appliances"